### PR TITLE
Some Tidy Up

### PR DESCRIPTION
**Pull request by Caleb-o**
I went through and did a little bit of tidy up, adding a couple of helper functions to reduce some repetitiveness. I don't have the ENVs, so I can't test it properly.

### CODE REVIEW

File: db/sqlite.go [hunk 1]
```diff
@@ -24,59 +24,52 @@ func InitDatabase() *Database {
...
+	statement.Close()
+
+	return
+}
+
 // InsertUser
 func (d *Database) InsertUser(id int, username string, displayName string) *User {
 	statement, err := d.db.Prepare(INSERT_USER)
-	defer statement.Close()
+	func() { _ = statement.Close() }()
```
Comment (SouLxBurN) [inline db/sqlite.go:72]: Doesn't this execute the Close() immediately? Why would this be better than defer?

File: db/sqlite.go [hunk 2]
```diff
@@ -24,59 +24,52 @@ func InitDatabase() *Database {
...
+	statement, err := db.Prepare(query)
+	if err != nil {
+		return
+	}
+
+	statement.Exec()
+	statement.Close()
+
+	return
+}
```
Comment (SouLxBurN) [inline db/sqlite.go:67]: I like this, getting rid of some of the repeated boilerplate. Thanks!

**Review by SouLxBurN — COMMENTED** [review]
Curious why remove the defer's for a `func() { .Close() }` Otherwise LGTM!

File: db/sqlite.go (hunk 1 above)
Comment (Caleb-o) [reply db/sqlite.go:72]: Actually, I think that was a replace issue, as it was meant to be `defer func() { _ = rows.Close() }()` and not just the function call. I'm fixing that now.

Comment (SouLxBurN) [reply db/sqlite.go:72]: That makes sense.

**Comment by Caleb-o** [comment]
Should be all good now

**Review by SouLxBurN — APPROVED** [review]

**Pull request merged**